src/main/webapp/WEB-INF/views/profile.jsp

```diff
@@ -17,6 +19,14 @@
             </h1>
         </div>
 
+        <div class="flex items-center justify-center" >
+            <c:forEach items="${badges.badges}" var="badge">
+                <img class="h-24 rounded-lg border-solid border-4 border-red-600 mx-4"
+                     src="<c:out value="${badge.imageUrl}"/>"
+                     title="<c:out value="${badge.title}"/>" />
```

Comment:
I am personally not a fan of the red border.
I also thought that the badges would look different, but we can talk about that on monday